packages/react-core/src/components/InputGroup/InputGroupText.tsx

```diff
@@ -2,35 +2,29 @@ import * as React from 'react';
 import styles from '@patternfly/react-styles/css/components/InputGroup/input-group';
 import { css } from '@patternfly/react-styles';
 
-export enum InputGroupTextVariant {
-  default = 'default',
-  plain = 'plain'
-}
```

Comment:
I'm not sure if we would want to remove this enum and the `variant` prop. Part of what the `plain` variant should be doing is removing that bottom border from `pf-c-input-group__item`. My comment below mentions it as well, but it looks like the cause of the padding issue might be the markup not matching Core. @srambach wdyt (including my other comment below)?

Reply:
I think we want to keep the option to make it plain or not, and I agree that the problem seems to be that the markup is slightly different as @thatblindgeye comments below.